plugin-management-cli/src/main/java/io/jenkins/tools/pluginmanager/cli/PluginParser.java

```diff
@@ -88,7 +90,7 @@
                         String version = versionObject == null ? "latest" : versionObject.toString();
                         Object urlObject = pluginSource.get("url");
                         String url;
-                        if (urlObject != null && urlValidator.isValid(urlObject.toString())) {
+                        if (urlObject != null && URL_VALIDATOR.isValid(urlObject.toString())) {
                             url = urlObject.toString();
                         } else {
                             url = null;
```

Comment:
perhaps a good idea to log invalid URLs?

Reply:
That would be a good idea